File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        
        unordered_set<int>st;
        for(int i=0;i<nums.size();i++){

            st.insert(nums[i]);

        }
        return !(nums.size()==st.size());



    }
};

File: Data Structures & Algorithms/find-pivot-index/submission-0.java
class Solution {
    public int pivotIndex(int[] nums) {
        
        int sum=Arrays.stream(nums).sum();
       
        int curSum=0;
        for(int i=0;i<nums.length;i++){
             
             if(curSum==(sum-curSum-nums[i])){
                return i;
             }
             curSum+=nums[i];


        }
        return -1;

    }
}